pkgs/tools/graphics/graph-easy/default.nix

```
@@ -1,7 +1,7 @@
 { lib, perlPackages, fetchurl }:
 
 perlPackages.buildPerlPackage {
-  pname = "Graph-Easy";
+  pname = "graph-easy";
```

```suggestion
  pname = "Graph-Easy";
```

This seems to be their self-spelling and in line with other perl packages.